var/.gitignore

```diff
@@ -1,6 +1,2 @@
 *
 .*
-!package
-!package/*
-!.htaccess
-!.gitignore
```

Comment:
I dont think we should touch this file as part of this

Reply:
Yes, sorry, accidental commit from another branch I was working on.. Will clean it up before merge.